cli/cobra_test.go

```diff
@@ -32,28 +32,29 @@ func TestVisitAll(t *testing.T) {
 	assert.DeepEqual(t, expected, visited)
 }
 
-func TestCommandVendor(t *testing.T) {
+func TestCommandVendorAndVersion(t *testing.T) {
```

Comment:
nit (not a show-stopper); 
```suggestion
func TestVendorAndVersion(t *testing.T) {
```

Reply:
Done